Makefile

```diff
@@ -0,0 +1,45 @@
+# See: https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63
+# For a list of valid GOOS and GOARCH values
+# Note: these can be overriden on the command line e.g. `make PLATFORM=<platform> ARCH=<arch>`
+PLATFORM=$(shell go env GOOS)
+ARCH=$(shell go env GOARCH)
+
+# build target, can be one of "production", "testing", "development"
+ENV=development
+WEBPACK_CONFIG=webpack/webpack.$(ENV).js
+
+.PHONY: help build-storybook build-client devops download-binaries tidy clean client-deps
+
+##@ Building
+
+build-storybook:
+	@yarn storybook:build
+
+build-client: ## Build the client
+	@export NODE_ENV=$(ENV) && yarn build:client --config $(WEBPACK_CONFIG)
+
+devops: clean download-binaries build-client ## Build the devops binary
+	@echo "Building the devops binary..."
+	@./build/build_binary_azuredevops.sh "$(PLATFORM)" "$(ARCH)"
+
```

Comment:
Can we add another section with a `test` target?

content basically @cd api && go test ./...

Reply:
Maybe we could add other target types for running and building portainer. e.g. the dev:client etc.
You could make it run the api and dev client in one command?

Reply:
we have a lot of those things in yarn, so do you want to move all server scripts to makefile? make sense...